codejam-virtual-keyboard/src/js/lib/create-element.js

```diff
@@ -0,0 +1,33 @@
+const is = require('./is');
+
+/** Create an instance of HTMLElement by tagName and set its some properties.
+ * This function can't proccess non primitive properties.
+ *
+ * @param {string} tagName
+ * @param {string} [className]
+ * @param {Object} [attrs]
+*/
+module.exports = function createElement(tagName, className, attrs) {
+  if (!attrs && is.object(className)) {
```

Comment:
Why you check that className is object if in jsdoc-description I see that is a string? Something is not clear here.